packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -663,7 +645,7 @@ export class MgtFileList extends MgtTemplatedComponent {
     return html`
       <div id="file-list-wrapper" class="file-list-wrapper" dir=${this.direction}>
         ${this.enableFileUpload ? this.renderFileUpload() : null}
-        ${this._isFullView ? html`<div class="title">${this.strings.filesSectionTitle}</div>` : null}
+        <div class="title">${this.strings.filesSectionTitle}</div>
```

Comment:
Because `renderFiles` is only ever called from `renderFullView` the `_isFullView` field was unnecessary

Reply:
@gavinbarron _isFullView is still necessary as the FileList component in itself has no title, this only becomes applicable when viewed as the files person-card section. In removing that hack, the FileList component has the title: Files

Reply:
Another suggestion would be give every section component a title property and have it used in renderCurrentSection method of person-card, right before the fullView is rendered. This would be different to the displayName currently used only in compactView.

Reply:
with the title property, would this be a read-only property and offload the rendering of the title to the component hosting the section?

If so, I like it.

Reply:
@Mnickii I've changed this implentation based on your feed back, it's in this commit: https://github.com/microsoftgraph/microsoft-graph-toolkit/pull/2125/commits/a2e27950242abd9ccc98bf3a314896e6c35dab9d